app/Repositories/Cts/ValidationPositionRepository.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Repositories\Cts;
+
+use App\Models\Cts\Member;
+use App\Models\Cts\ValidationPosition;
+use Carbon\Carbon;
+use Illuminate\Support\Collection;
+
+class ValidationPositionRepository
+{
+    public function findByPositionId(int $id)
+    {
+        return ValidationPosition::find($id)->first();
+    }
+
+    public function findByPosition(string $position)
+    {
+        return ValidationPosition::where('position', 'like', "%{$position}%")->first();
+    }
+
+    public function getValidatedMembersFor(ValidationPosition $validationPosition)
+    {
+        return $this->formatValidatedMembers($validationPosition->members);
+    }
+
+    private function formatValidatedMembers($validatedMembers)
+    {
+        $formattedMembers = [];
```

Comment:
This entire function can be simplfied into:
```php
return collect($validatedMembers)->map(function ($member) {
    return ['id' => $member->cid, 'name' => $member->name];
});
```
Let me know if you have any questions.

Reply:
Simplified even further in commit 83aa7d9b